rita/src/rita_common/rita_loop/mod.rs

```diff
@@ -126,6 +126,22 @@ impl Handler<Tick> for RitaLoop {
                 }),
         );
 
+        let start = Instant::now();
+        Arbiter::spawn(
+            TunnelManager::from_registry()
+                .send(TriggerGC(Duration::from_secs(15)))
```

Comment:
As mentioned few minutes (i.e. 15min) might be more user friendly. Or even a configuration setting with a default value.

Reply:
yes this needs to take a configured duration